kubeflow/common/iap-ingress/base/config-map.yaml

```diff
@@ -178,11 +178,11 @@ data:
         gcloud --project=${PROJECT} compute health-checks update http ${HEALTH_CHECK_URI} --request-path=${HEALTHCHECK_PATH}
       else
         # /healthz/ready is the health check path for istio-ingressgateway
-        echo Running health checks update ${HEALTH_CHECK_URI} with /healthz/ready
-        gcloud --project=${PROJECT} compute health-checks update http ${HEALTH_CHECK_URI} --request-path=/healthz/ready
```

Comment:
Yep, makes total sense.

Reply:
I thought about that.
I wanted this script to change the minimum possible. I think that the blueprint manifests should be the source of truth.
We only need to patch the port because it is randomly assigned, ideally we would set it to a fixed number.

I think that the health check resource is actually created from the `BackendConfig`.
When we fetch `HEALTH_CHECK_URI` :

```bash
HEALTH_CHECK_URI=$(gcloud compute --project=${PROJECT} health-checks list --filter=name~${BACKEND_NAME} --uri)
```
Aren't we just fetching the same value we defined in the yaml ?

Reply:
Hi @zijianjoy ,

I am still confused.
Isn't the health check created and configured based on the `BackendConfig`'s spec.healthCheck definition ?

```yaml
apiVersion: cloud.google.com/v1beta1
kind: BackendConfig
metadata:
  name: iap-backendconfig
spec:
  healthCheck:
    port: 31223
    type: HTTP
    requestPath: /healthz/ready # <===
```
I `requestPath` is already set to `/healthz/ready` in the yaml why would we need to reset it using gcloud ?
Am I missing something ?

Reply:
In included an patch op for the requestPath